### PR TITLE
Implemented PPNP model

### PR DESCRIPTION
**Pull request by TiagoMAntunes**
Hi,

I've implemented the[ Predict then Propagate: Graph Neural Networks meet Personalized PageRank](https://arxiv.org/pdf/1810.05997.pdf) model in CogDL.

A note, the model basically has 2 modes, which depends or not on a PowerIteration step. They are ppnp and appnp. It's the same base, so they are switched in the `--propagation` flag.

I've also only added _appnp_ to the table as in the paper it shows that it's better than _ppnp_.

### CODE REVIEW

**Comment by TiagoMAntunes** [comment]
Also, I noticed that we can give multiple datasets as input, but other arguments aren't possible. If I for example wanted to use `--propagation ppnp appnp` I'd have to run it 2 times as it would give me an error saying `appnp argument not recognized`. Is this possible to change? If yes I'll open an issue

**Comment by cenyk1230** [comment]
Great work. You can see our code in the `scripts/train.py` and we only split three arguments (i.e., dataset, model, seed) to different running variants. Other arguments, such as model-specific arguments, are somehow hard to handle.
